library/std/src/os/hermit/io/mod.rs

```diff
@@ -1,13 +1,4 @@
-#![stable(feature = "os_fd", since = "1.66.0")]
+#![stable(feature = "rust1", since = "1.0.0")]
 
-mod net;
-#[path = "../../fd/owned.rs"]
-mod owned;
-#[path = "../../fd/raw.rs"]
-mod raw;
-
-// Export the types and traits for the public API.
-#[stable(feature = "os_fd", since = "1.66.0")]
-pub use owned::*;
-#[stable(feature = "os_fd", since = "1.66.0")]
-pub use raw::*;
+#[stable(feature = "rust1", since = "1.0.0")]
+pub use crate::os::fd::*;
```

Comment:
I don't think this is right as `std::os::fd` only existings since 1.66.

Reply:
Actually never mind, this is `std::os::hermit`, so it's probably fine (?)

Reply:
I am not sure. I synchronized it with `std/src/os/unix/io/mod.rs`. But I don't know, if I used the correct version.